### PR TITLE
Initial template for classes

### PR DESCRIPTION
**Pull request by scifi6546**
Wrote initial template. Not too experienced with c++ polymorphism so there may be some mistakes.

### CODE REVIEW

File: src/cps.hpp [hunk 1]
```diff
@@ -4,11 +4,30 @@
 
 #ifndef CS372_CPS_CPS_HPP
 #define CS372_CPS_CPS_HPP
+#include <iostream>
 
+class Shape {
+    public:
+    ~Shape();
+    Shape();
+    virtual std::iostream draw()=0;
```
Comment (Jlrine2) [inline src/cps.hpp:13]: Should this be `virtual std::iostream draw() {};`

File: src/cps.hpp [hunk 2]
```diff
@@ -4,11 +4,30 @@
...
 #define CS372_CPS_CPS_HPP
+#include <iostream>
 
+class Shape {
+    public:
+    ~Shape();
+    Shape();
+    virtual std::iostream draw()=0;  
+};
+class Polygon: Shape{
```
Comment (Jlrine2) [inline src/cps.hpp]: Tried using this code, looks good but I needed to add public to each of these in order to be able to use them

```c++
class Polygon: public Shape {};
```

**Review by Jlrine2 — CHANGES_REQUESTED** [review]

File: src/cps.hpp (hunk 2 above)
Comment (Jlrine2) [reply src/cps.hpp]: see: https://github.com/Jlrine2/CS372-CPS/pull/5

**Review by scifi6546 — COMMENTED** [review]
That looks a lot better

**Review by Jlrine2 — APPROVED** [review]

**Pull request merged**